bridge/hardhat/helper/chains.js

```diff
@@ -0,0 +1,49 @@
+const ETHEREUM_MAIN_NET_CHAIN_ID = 1;
+const RSK_MAIN_NET_CHAIN_ID = 30;
+const RSK_TEST_NET_CHAIN_ID = 31;
+const KOVAN_TEST_NET_CHAIN_ID = 42;
+const BSC_MAIN_NET_CHAIN_ID = 56;
+const BSC_TEST_NET_CHAIN_ID = 97;
+const GANACHE_DEV_MIRROR_CHAIN_ID = 5776;
+const GANACHE_DEV_NET_CHAIN_ID = 5777;
+const HARDHAT_TEST_NET_CHAIN_ID = 1999;
+
+/** Network example
```

Comment:
shouldn't this example be in the hardhat config file?

Reply:
It feels kind of weird to import the hardhat conf in other files

Reply:
agree, besides hardhat config is getting big

Reply:
no, I mean the comment - we only configure networks in the hardhat config, wouldn't it make sense for this comment to be in the hardhat config file?
I mean this comment
```json
  config: {
    accounts: 'remote',
    live: false,
    network_id: 5777,
  },
  saveDeployments: false,
  tags: (2) ['integrationTest', 'local'],
  live: false,
  name: 'development'
```

Reply:
It is just like an example where the network param is used, so we can get some information without having to debug 

If we had the scripts in typescript we are not going to need the comments